systemtest/src/main/java/io/strimzi/systemtest/Constants.java

```diff
@@ -13,7 +13,7 @@
     long TIMEOUT_FOR_DEPLOYMENT_CONFIG_READINESS = Duration.ofMinutes(7).toMillis();
     long TIMEOUT_FOR_RESOURCE_CREATION = Duration.ofMinutes(5).toMillis();
     long TIMEOUT_FOR_SECRET_CREATION = Duration.ofMinutes(2).toMillis();
-    long TIMEOUT_FOR_RESOURCE_READINESS = Duration.ofMinutes(12).toMillis();
+    long TIMEOUT_FOR_RESOURCE_READINESS = Duration.ofMinutes(14).toMillis();
```

Comment:
Out of curiosity why this change?

Reply:
I find out, that deployment of Kafka Connect sometimes needs approximately 13 minutes to deploy but on the other hand somethings it takes only 1 minute.

Reply:
Unfortunately, it's a limitation of one from you QE environments and we can't do much with that. This is probably the easiest solution.